src/Translator.php

```diff
@@ -119,10 +119,20 @@ private function scanDir(string $dir): array
         return array_diff(scanDir($dir), ['.', '..']);
     }
 
+    private function getCmsMajor(): string
```

Comment:
```suggestion
    private function getCmsFrameworkMajor(): string
```
or
```suggestion
    private function getFrameworkMajor(): string
```
or 
query `composer show silverstripe/cms` ?

Reply:
Updated